service/src/main/java/uk/gov/hmcts/reform/fpl/config/HmctsCourtLookupConfiguration.java

```diff
@@ -33,10 +34,12 @@ public Court getCourt(String localAuthorityCode) {
     public static class Court {
         private final String name;
         private final String email;
+        private final String location;
 
-        public Court(String name, String email) {
+        public Court(String name, String email, String location) {
```

Comment:
Seeing as we are using lombok couldn't this be replaced with the `@Data` annotation on the class?